### ISSUE TITLE
how to add effect filters in streaming?

### ISSUE THREAD

**Issue by luigbren** [NodeMedia/react-native-nodemediaclient#87]
I would like to add in live stream filters of effects in the face as it uses the android camera, instagram, snapchat, example..
[https://cdn.guidingtech.com/imager/assets/188669/Screenshot_20171229-123001_7c4a12eb7455b3a1ce1ef1cadcf29289.jpg?1562327214](url)

[https://cdn.guidingtech.com/imager/assets/188670/Screenshot_20171229-144711_7c4a12eb7455b3a1ce1ef1cadcf29289.jpg?1562327214](url)

**Comment by BinniZenobioCordovaLeandro** [comment]
Thats true, please can indicate how is possible add effects or what module can i inject to this module that is amazing.

**Comment by BinniZenobioCordovaLeandro** [comment]
Anyone used this module to add effects into the streaming?

**Comment by adnandothussain** [comment]
Looking for the same thing!
